internal/controller/builder/cue/restore_full_backup_job.cue

```diff
@@ -42,7 +42,7 @@ backupTool: {
 	spec: {
 		image: string
 		physical: {
-			restoreCommands: [string]
```

Comment:
have you scanned all like-wise issues?

Reply:
No echo ?

Reply:
that's kind of mistake I made when wrote this cue template, no similar error in other templates as I checked.